README.md

```diff
@@ -1,138 +1,52 @@
+>
+> The old readme file can be found here: [old readme](oldReadme.md)
 
-The action consists of 2 stages: `Upload` and `Deploy`.
+## Migration to .NET Tools
 
-## Stages
+Instead of using the GitHub action, we are now using .NET tools. These are more flexible as they can be used on any commonly used platform like GitHub, GitLab, Azure DevOps, ...
 
-### Upload
+The GitHub Action did 3 things in 1 which wasn't always what was wanted by our users:
 
-This stage creates an artifact and uploads it to dedicated storage in the cloud. The output of this stage will be the ID of the artifact, which can be used in the deploy stage.
+- Creating a package
+- Uploading a package to Catalog
+- Deploying a package to a DataMiner
 
-### Deploy
+These 3 steps have been converted into their own .NET tool and can be used separately depending on the scenario.
 
-This stage deploys the artifact from the artifact storage to your cloud-connected DataMiner System.
-
-## Limitations
-
-This action currently only supports the creation of artifacts with Automation scripts.
-
-## Inputs
-
-### `api-key`
-
-**Required**. The API key generated in the [DCP Admin app](https://admin.dataminer.services) to authenticate to a certain DataMiner System. E.g. `${{ secrets.NAME_OF_YOUR_APIKEY_SECRET }}`. For more information about creating a key, refer to the [DataMiner documentation](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_DCP_keys.html).
-
-### `solution-path`
-
-**Required**. The path to the .sln file of the solution. At present, only DataMiner Automation Script solutions are supported. E.g. `'./Example/AutomationScript.sln'`. Required for stages `'Upload'` and `'All'`.
-
-### `artifact-name`
-
-**Optional**. The chosen name for the artifact. E.g. `'MyPackageName'`. Required for stages `'Upload'` and `'All'`.
-
-### `version`
-
-**Optional**.
-The version number for the artifact. Only required for a release run. Format A.B.C for a stable release or A.B.C-text for a pre-release. E.g. `'1.0.1'`. Required for stages `'Upload'` and `'All'` if no build-number was provided instead.
-
-### `timeout`
-
-**Optional**. The maximum time spent waiting for the deployment to finish, in seconds. Default '900'. E.g. `'300'`.
-
-### `stage`
-
-**Optional**. The stage of the action to run. Options are: `'Upload'`, `'Deploy'` and `'All'`. Default: 'All'.
-
-### `artifact-id`
-
-**Optional**. The private artifact to deploy. This is only needed when 'stage' is `'Deploy'`.
-
-### `build-number`
-
-**Optional**.
-The build number of a workflow run. Only required for a development run. Required for stages `'Upload'` and `'All'` if no version was provided instead.
-
-## Outputs
-
-### `artifact-id`
-
-The ID of the private artifact that has been deployed. This is only filled in for stages `'Upload'` and `'All'`.
-
-## Example usage
-
-### All stages at once
+Below is an example of a full migration of the GitHub action:
```

Comment:
## Transition to .NET Tools Integration

Our workflow has evolved from relying solely on GitHub actions to embracing the versatility of .NET tools. This transition offers enhanced flexibility, enabling seamless integration across various widely utilized platforms such as GitHub, GitLab, Azure DevOps, and more.

Previously, the GitHub Action encompassed a bundled approach, performing three distinct tasks:

1. Generating a package
2. Uploading the package to the Catalog
3. Deploying the package to a DataMiner

However, this bundled functionality didn't always align with the specific needs of our users. In response, we have modularized these tasks into individual .NET tools, allowing for tailored usage based on the unique requirements of each scenario.

Below, we present an example detailing the migration process from the GitHub action: